# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/Http3Session.h
# Commit: b45bd4ea0000
# Full Hash: b45bd4ea0000e64e4b159cac01d09519d73166f7
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-17 09:39:01
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r 705439f1d269 -r b45bd4ea0000 netwerk/protocol/http/Http3Session.h
--- a/netwerk/protocol/http/Http3Session.h	Wed Nov 16 16:37:29 2022 +0000
+++ b/netwerk/protocol/http/Http3Session.h	Wed Nov 16 16:37:29 2022 +0000
@@ -7,17 +7,18 @@
 #ifndef Http3Session_H__
 #define Http3Session_H__
 
+#include "HttpTrafficAnalyzer.h"
+#include "mozilla/UniquePtr.h"
+#include "mozilla/WeakPtr.h"
+#include "mozilla/net/NeqoHttp3Conn.h"
+#include "nsAHttpConnection.h"
+#include "nsDeque.h"
 #include "nsISupportsImpl.h"
 #include "nsITimer.h"
 #include "nsIUDPSocket.h"
-#include "mozilla/net/NeqoHttp3Conn.h"
-#include "nsAHttpConnection.h"
 #include "nsRefPtrHashtable.h"
+#include "nsTHashMap.h"
 #include "nsWeakReference.h"
-#include "HttpTrafficAnalyzer.h"
-#include "mozilla/UniquePtr.h"
-#include "mozilla/WeakPtr.h"
-#include "nsDeque.h"
 
 /*
  * WebTransport